js/modules/loki_snode_api.js

```diff
@@ -348,19 +348,28 @@ class LokiSnodeAPI {
     const guards = _.shuffle(this.guardNodes);
 
     // Create path for every guard node:
-
-    // Each path needs 2 nodes in addition to the guard node:
-    const maxPath = Math.floor(Math.min(guards.length, otherNodes.length / 2));
+    const needPaths = window.lokiFeatureFlags.onionRequestPaths - 1;
```

Comment:
`needPaths`? How about `needNodesPerPath`? Also `onionRequestPaths` below should be called something like `nodesPerPath`.

Reply:
Going with `onionRequestHops` and `nodesNeededPerPaths`